spec/convertion_by_format/check_open_docx_by_screen_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require 'spec_helper'
+require './spec/spec_helper'
```

Comment:
changing path to `spec_helper` is not right, `require 'spec_helper'` should work